src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingPropertySymbol.cs

```diff
@@ -187,6 +187,16 @@ private ImmutableArray<PropertySymbol> RetargetExplicitInterfaceImplementations(
             return builder.ToImmutableAndFree();
         }
 
+        internal override PropertySymbol ExplicitlyOverriddenClassProperty
+        {
+            get
+            {
+                return _underlyingProperty.OverriddenProperty is PropertySymbol overridden
```

Comment:
>OverriddenProperty [](start = 43, length = 18)

This looks like a bug. Should this be ```ExplicitlyOverriddenClassProperty```? It looks like there is a test gap as well.

Reply:
No, this is intended and necessary for correctness. Most of the tests would break if changed as suggested. Having said that, this method is removed later in this PR.

---
In reply to: [424665880](https://github.com/dotnet/roslyn/pull/44025#discussion_r424665880) [](ancestors = 424665880)

Reply:
> No, this is intended and necessary for correctness. Most of the tests would break if changed as suggested. Having said that, this method is removed later in this PR.

The method is still present in the last iteration (iteration 30)


---
In reply to: [425945737](https://github.com/dotnet/roslyn/pull/44025#discussion_r425945737) [](ancestors = 425945737,424665880)

Reply:
The virtual method that was present in iteration 13 no longer exists as a virtual method. This method replaces the override. The implementation is as intended.

---
In reply to: [441976888](https://github.com/dotnet/roslyn/pull/44025#discussion_r441976888) [](ancestors = 441976888,425945737,424665880)